src/promptflow/promptflow/_sdk/_service/entry.py

```diff
@@ -135,7 +136,7 @@ def entry(command_args):
     if args.action == "start":
```

Comment:
why we need to use the with pattern? 
I orginally think we can do custom two log function call for action start/complete?

Reply:
updated